specification/gp-connect-access-record-fhir.yaml

```diff
@@ -14732,10 +14739,12 @@ paths:
             An error occured as follows:
 
 
-            | HTTP Status | Spine error code  | Description                                                                 |
-            | ----------- | ----------------  | -----------                                                                 |
-            | 404         | NO_RECORD_FOUND   | A document could not be found with the document id provided                 |
-            | 404         | NO_RECORD_FOUND   | The document could not be retrieved due to it exceeding the file size limit |
+            | HTTP Status | Spine error code  | Description                                                                              |
+            | ----------- | ----------------  | -----------                                                                              |
+            | 404         | NO_RECORD_FOUND   | A document could not be found with the document id provided                              |
+            | 404         | NO_RECORD_FOUND   | The document could not be retrieved due to it exceeding the file size limit              |
```

Comment:
@ashcox  That is a good question. My understanding of the reasoning might not be 100% complete because I wasn't here it was decided, but to my knowledge they were mapped to the most relevant code available.